test/extended/networking/isolation.go

```diff
@@ -7,13 +7,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// This test requires a network plugin that supports namespace isolation.
-// NOTE: if you change the test description, update networking.sh too!
 var _ = Describe("[networking] network isolation plugin", func() {
 	f1 := e2e.NewFramework("net-isolation1")
 	f2 := e2e.NewFramework("net-isolation2")
 
-	It("should prevent communication between pods in different namespaces", func() {
+	var expectation string
+	if isMultitenant() {
```

Comment:
I think it would be preferable to have the test methods defined statically and skipped as appropriate.  It might be confusing to those analyzing test results if the list of tests varied according to the setup being tested. Consider instead creating a parametrized helper method that can be called from 2 tests, e.g.

    multitenant := isMultitenant()
    It("should prevent...") {
        if multitenant {
            checkPodIsolation(multitenant)
        } else {
            e2e.Skipf("Tenant isolation is not supported")
        }
    }
    It("should allow...") {
        ...
    }